context_test.go

```diff
@@ -0,0 +1,34 @@
+package gb
+
+import (
+	"runtime/debug"
+	"strings"
+	"testing"
+)
+
+func testImportCycle(pkg string, t *testing.T) {
```

Comment:
you can reuse the table in `build_test.go`, 

add entries for "cycle0", etc, and include the error you expect.

That should be good enough